static/src/javascripts-legacy/projects/commercial/modules/article-body-adverts.js

```diff
@@ -46,12 +50,16 @@ define([
             // we must wait for DFP to return, since if the merch
             // component is empty, it might completely change the
             // positions where we insert those MPUs.
-            im.then(waitForMerch).then(addInlineAds);
+            im.then(waitForMerch).then(addInlineAds).then(moduleEnd);
             return im;
         }
 
-        addInlineAds();
+        addInlineAds().then(moduleEnd);
         return Promise.resolve(true);
+
+        function moduleEnd() {
+            performanceLogging.moduleEnd(moduleName);
```

Comment:
Q: Is there a reason for the use of `moduleEnd()` over `.bind` in this case?

Other modules are using: 
`.then(performanceLogging.moduleEnd.bind(null, moduleName))` 🤔